script.js

```diff
@@ -1,10 +1,25 @@
 jQuery(function () {
 
+    var aContainer = jQuery('.plugin-acknowledge-assign');
```

Comment:
we usually prefix variables that contain a jQuery instance with `$`

Reply:
right